toolchains/mxe/packages/toolchain/build.sh

```diff
@@ -23,12 +26,12 @@ cd "${MXE_DIR}"/
 
 # Override PREFIX on command line and not in settings.mk because else it gets overriden too late
 
-make PREFIX="${MXE_PREFIX_DIR}" check-requirements
+make $MAKEFLAGS check-requirements
```

Comment:
Could you not make use of the unquoted variable and add everywhere -j$JOBS? I think it's more clear about what happens.

Reply:
Done